### PR TITLE
Fix guides links

### PR DESCRIPTION
**Pull request by neojp**
This PR addresses #1440 for Ember `v3.21.0`.

## Process

1. Ran script from #1592
2. Manually checked updated links
3. Ran `npm test:node-local` to check all external links automatically

## Result

```
node update-version-links.js ../guides/v3.21.0 3.21

> Welcome to the automated process of updating the Guides version links
> 
> Found: 168 files
> Updated: 34 files


npm run test:node-local

> check all external links in markdown files
> ...
> 168 passing (4m)
```

### CODE REVIEW

File: guides/v3.21.0/index.md [hunk 1]
```diff
@@ -7,13 +7,13 @@ With the plethora of libraries readily available for front-end development, some
 		<div class="shape shape--dark"></div>
     <div class="shape shape--accent"></div>
     <div class="shape shape--light"></div>
-		<a href="https://guides.emberjs.com/release/getting-started/">Ember.js</a>
+		<a href="https://guides.emberjs.com/v3.21.0/getting-started/">Ember.js</a>
```
Comment (ijlee2) [inline guides/v3.21.0/index.md:10]:
```suggestion
		<a href="./getting-started/">Ember.js</a>
```

File: guides/v3.21.0/index.md [hunk 2]
```diff
@@ -7,13 +7,13 @@ With the plethora of libraries readily available for front-end development, some
...
 	</li>
 	<li class="list-item-card">
 		<div class="shape shape--dark"></div>
     <div class="shape shape--accent"></div>
     <div class="shape shape--light"></div>
-		<a href="https://guides.emberjs.com/release/models/">Ember Data</a>
+		<a href="https://guides.emberjs.com/v3.21.0/models/">Ember Data</a>
```
Comment (ijlee2) [inline guides/v3.21.0/index.md:16]:
```suggestion
		<a href="./models/">Ember Data</a>
```

File: guides/v3.21.0/index.md [hunk 3]
```diff
@@ -24,5 +24,5 @@ With the plethora of libraries readily available for front-end development, some
 		<div class="shape shape--dark"></div>
     <div class="shape shape--accent"></div>
     <div class="shape shape--light"></div>		
-		<a href="https://guides.emberjs.com/release/ember-inspector/">Ember Inspector</a></li>
+		<a href="https://guides.emberjs.com/v3.21.0/ember-inspector/">Ember Inspector</a></li>
```
Comment (ijlee2) [inline guides/v3.21.0/index.md:27]:
```suggestion
		<a href="./ember-inspector/">Ember Inspector</a></li>
```

**Review by ijlee2 — CHANGES_REQUESTED** [review]
I think that 3 internal links in `index.md` shouldn't have been caught by the script that you wrote. For internal links, we can use a relative path (as shown in my suggestions).

Can you **push 1 commit** to update `index.md` for `v3.16` to `v3.21` as well as `release`? We're having trouble with Travis backlogs so I want to make sure that we push few commits and make few pull requests.

File: guides/v3.15.0/object-model/enumerables.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ where available.
       <div class="cta-note-heading">Zoey says...</div>
       <div class="cta-note-message">
         It is best practice to
-        <a href="https://guides.emberjs.com/release/configuring-ember/disabling-prototype-extensions/">
```
Comment (ijlee2) [inline guides/v3.15.0/object-model/enumerables.md:23]: It looks like the link points to `release` for `v3.8` to `v3.14` as well. Can you push an extra commit where you use a relative path instead? I haven't checked locally but the relative path may be,

```
../configuring-ember/disabling-prototype-extensions/
```

If not,

```
../../configuring-ember/disabling-prototype-extensions/
```

**Review by ijlee2 — APPROVED** [review]
Hi, @neojp. I manually checked the changes and they looked great.

I left 1 additional suggestion for relative URL. While we can merge the PR as is now, I think it'd be nice to add the relative URL in `object-model/enumerables.md` for `v3.8` to `v3.15` before merging the PR.

**Comment by neojp** [comment]
@ijlee2 I have updated the links you mentioned.

**Pull request merged**